CRM/Core/BAO/UFField.php

```diff
@@ -221,24 +219,6 @@ public static function add(&$params, $ids = array()) {
     }
 
     $ufField->phone_type_id = CRM_Utils_Array::value(3, $params['field_name'], 'NULL');
-    $ufField->listings_title = CRM_Utils_Array::value('listings_title', $params);
-    $ufField->visibility = CRM_Utils_Array::value('visibility', $params);
-    $ufField->help_pre = CRM_Utils_Array::value('help_pre', $params);
-    $ufField->help_post = CRM_Utils_Array::value('help_post', $params);
-    $ufField->label = CRM_Utils_Array::value('label', $params);
-    $ufField->is_required = CRM_Utils_Array::value('is_required', $params, FALSE);
```

Comment:
Thanks for pointing that out. I believe copyValues function will set the value to NULL rather than FALSE. I'll look into it ASAP.